packages/expo-camera/ios/Next/CameraViewNext.swift

```diff
@@ -417,11 +415,10 @@ public class CameraViewNext: ExpoView, EXCameraInterface, EXAppLifecycleListener
 
     takenImage = ExpoCameraUtils.crop(image: takenImage, to: croppedSize)
 
-    guard let path = fileSystem?.generatePath(
-      inDirectory: fileSystem?.cachesDirectory.appending("/Camera"),
-      withExtension: ".jpg") else {
-      return
-    }
+    let fileUtils = FileSystemUtilities(appContext: appContext)
+    let path = fileUtils.generatePath(
+      in: appContext?.config.cacheDirectory?.appendingPathComponent("Camera"),
```

Comment:
Looks like we only use `generatePath` with the cache directory, so maybe it makes sense to have `generatePathInCache(_ appContext: AppContext, directory: String, extension: String)` where the directory is just `"Camera"` and the implementation will take care of getting the cache directory from the app context.